README.md

```diff
@@ -10,4 +10,4 @@ Caesar Saleh
 Youssef Soliman
 Sarah Xu
 Sataphon 
-Valerie Yip
\ No newline at end of file
+Valerie Yip
```

Comment:
This shouldn't be in this PR

Reply:
My mistake, branched from an older commit by mistake and thought a merge would fix it but needed to do a single file checkout instead.